api/api.js

```diff
@@ -37,7 +37,7 @@ schedule.scheduleJob('0 3 * * *', function(){
     var gzip = zlib.createGzip();
     var today = new Date();
     var wstream = fs.createWriteStream('/pheromon/data/backups/' + today.getDay() + '.txt.gz');
-    var proc = spawn('pg_dump', ['-p', process.env.DB_PORT_5432_TCP_PORT, '-h', process.env.DB_PORT_5432_TCP_ADDR, '-U', process.env.POSTGRES_USER, '-d', process.env.POSTGRES_USER, '-w']);
+    var proc = spawn('pg_dump', ['-p', process.env.POSTGRES_PORT, '-h', process.env.POSTGRES_HOST, '-U', process.env.POSTGRES_USER, '-d', process.env.POSTGRES_USER, '-w']);
```

Comment:
Per discussion with @4rzael, this was done because `api: net: host` in docker-compose is incompatible with `link` (which makes sense). The solution is to create a new docker service that handles only the reverse SSH business (and revert the api service and code at this line to where it was). To be done in an upcoming commit.